### PR TITLE
revise the quantize-ort to quantize crnn into int8 with high accuracy

### PR DESCRIPTION
**Pull request by Charles-258**

### CODE REVIEW

File: tools/quantize/quantize-ort.py [hunk 1]
```diff
@@ -29,9 +29,10 @@ def get_next(self):
 
     def get_calibration_data(self, image_dir):
         blobs = []
+        supported = ["jpg", "JPG", "png", "PNG"]  # 支持的文件后缀类型
```
Comment (zihaomu) [inline tools/quantize/quantize-ort.py:32]: Please change the comment to English.

**Review by zihaomu — APPROVED** [review]
Thanks for your contribution! 👍 

Please take look the following comment. And don't forget to squash the commits into 1 commit

**Comment by Charles-258** [comment]
您好，我是王仡垚，您的邮件我已经收到，我会尽快回复。

File: tools/quantize/quantize-ort.py [hunk 2]
```diff
@@ -29,9 +29,10 @@ def get_next(self):
 
     def get_calibration_data(self, image_dir):
         blobs = []
+        supported = ["jpg", "JPG", "png", "PNG"]  # supported file suffix
```
Comment (fengyuentau) [inline tools/quantize/quantize-ort.py:32]: use `.lower()` and keep only one instance for each format.

File: tools/quantize/quantize-ort.py [hunk 3]
```diff
@@ -55,13 +56,13 @@ def __init__(self, model_path, calibration_image_dir, transforms=Compose(), per_
 
     def check_opset(self, convert=True):
         model = onnx.load(self.model_path)
-        if model.opset_import[0].version != 11:
-            print('\tmodel opset version: {}. Converting to opset 11'.format(model.opset_import[0].version))
-            # convert opset version to 11
-            model_opset11 = version_converter.convert_version(model, 11)
+        if model.opset_import[0].version != 13:
```
Comment (fengyuentau) [inline tools/quantize/quantize-ort.py:59]: which operator in quantized crnn needs opset 13?

Comment (zihaomu) [reply tools/quantize/quantize-ort.py:59]: @Charles-258 , please take a look.

Comment (Charles-258) [reply tools/quantize/quantize-ort.py:59]: > which operator in quantized crnn needs opset 13?

If quant_format=QuantFormat.QDQ and per_channel=True, we need to convert onnx into opset 13.
5MZ31C}UEF4`S SP%4H54W4

In other conditions, opset 11 is enough.

Comment (zihaomu) [reply tools/quantize/quantize-ort.py:59]: Looks like the difference is the `Quantizelinear` and `DequantizeLinear` which are very important in the QDQ mode. Quantizelinear of opset 13 has added a new attribute, [more details](https://github.com/onnx/onnx/blob/main/docs/Operators.md#quantizelinear).

**Comment by fengyuentau** [comment]
It is a bad move merging with `if model.opset_import[0].version != 13:`, which means all of our quantized models need to be re-generated. It should be special-cased for the crnn models.

**Pull request merged**